README.md

```diff
@@ -14,6 +14,7 @@ Then, open it in your editor of your choice, and set the following 3 variables:
 username = "Your username"        # The username you would usually use for My Manchester
 password = "Your password"        # The accompanying password
 base_dir = "~/Documents/Lectures" # Where to download files to
+concurrent_downloads = 8          # How many podcasts to download concurrently (Increases speed)
```

Comment:
Use simultaneously instead of concurrently for the description

Reply:
Concurrent is the correct term here, as the downloads are overlapping over a period of time, whereas simultaneously refers to a occurring at the same instant.